src/modules/logger/logger.cpp

```diff
@@ -518,6 +520,29 @@ void Logger::run()
 {
 	PX4_INFO("logger started (mode=%s)", configured_backend_mode());
 
+	bool boot_logging_bat_only = false;
```

Comment:
Can you move the code below into a separate method, so that you get:
```
const bool disable_boot_logging = getDisableBootLogging();
```

And move that down directly before it's used?

Reply:
Alright, will do